CHANGELOG.md

```diff
@@ -3,6 +3,97 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.0.0] / x August 2018
+
+We are very happy to announce a new major version of mustache.js. We want to be very careful not to break projects
+out in the wild, and adhering to [Semantic Versioning](http://semver.org/) we have therefore cut this new major version.
+
+The changes introduced will likely not require any actions for most using projects, the things to look out for that
```

Comment:
Minor grammatical nitpick: comma splice; there should be either a period or semicolon between "projects" and "the things", rather than a comma.